package.json

```diff
@@ -62,7 +62,14 @@
     "samples-test": "npm link && cd samples/ && npm link @google-cloud/storage && npm test && cd ../",
     "system-test": "repo-tools test run --cmd mocha -- system-test/ --timeout 600000",
     "all-test": "npm test && npm run system-test && npm run samples-test",
-    "generate-scaffolding": "repo-tools generate all && repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json"
+    "generate-scaffolding": "repo-tools generate all && repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json",
+    "check": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check"
```

Comment:
There is a very low chance this will work.  I would get rid of this line, and open an issue (if I haven't already) to turn `gts check` back on.